lib/src/slices/user_referral/ui/user_referral_view_code.dart

```diff
@@ -36,8 +35,7 @@ class UserReferralViewCode extends StatelessWidget {
             Container(
                 margin: EdgeInsets.only(
                     left: 2.w, top: 1.5.h, right: 1.h, bottom: 1.5.h),
-                child: Text(
-                    service.getCode(Provider.of<LoginFlowService>(context)),
+                child: Text(service.model.code ?? service.getCode(),
```

Comment:
shouldn't this just call service.getCode() which if it has the code it returns if it doesn't it shows "", otherwise this could get stuck in an infinite loop of